src/hooks/useTheme.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from "react";
+
+import { useLocalStorage } from "./useLocalStorage";
+
+export const useTheme = () => {
+  const [isDarkMode, setIsDarkMode] = useLocalStorage<"true" | "false">(
```

Comment:
I believe that we should store "theme" value in local storage not just if it's dark or not. Btw, we should also handle system theme by default

Reply:
Good point, also maybe we should use dropdown for the theme in settings. By default, it would be set to `system`.